examples/docs_beta_snippets/docs_beta_snippets_tests/ensure_snapshot_deps.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
-which tree || apt-get install tree
+# Validate that tree is installed, install it if not (for bk purposes)
+(which tree > /dev/null && echo "tree command found") ||
```

Comment:
I've seen people recommend avoiding `which` for checking if commands are present, since it's an external process and there are built-ins. I typically opt for `command -v`.

https://stackoverflow.com/a/677212

Not a blocker, but thought I'd share!

Reply:
Neat, easy enough to sub out so will update, thanks for the heads up